.github/workflows/localnet.yaml

```
@@ -0,0 +1,43 @@
+jobs:
+  setup-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Test Matrix
+        id: set-matrix
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2038:warning:1:13: Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames

(shellcheck)